### DISCUSSION TITLE
SwiftUSD - iOS enabled

### DISCUSSION THREAD

**Discussion by SimonBenAnderson** [wabiverse/Kraken#10] [Q&A] [answered]
Hi Wabiverse Team,

I was wondering if the SwiftUSD binding that you have work for the iOS environment?

As I have been looking for Swift OpenUSD bindings for an app

Thanks,
Simon

**Comment by furby-tm** [comment, marked as answer]
Hello @SimonBenAnderson! Thank you for your interest in SwiftUSD.

Currently, **iOS** remains untested and unattempted. However, it's worth noting that the Python, Boost, and MoltenVK `*.xcframework` dependencies are compatible with **iOS**, **visionOS**, and **macOS**.

To the best of my knowledge, **iOS** should behave similarly to **macOS**. Still, I want to mention that I need to implement the **Pixar.Pcp** and **Pixar.Usd** libraries as a minimum requirement for authoring scene description in Swift before this becomes usable for your development needs.

Once I've completed these two libraries, I'll ensure that **iOS** and **visionOS** builds work as expected. Afterward, I'll focus on bringing **Linux** in line with the Apple platforms.

I estimate that you should be able to minimally begin to integrate SwiftUSD into your **iOS** project as soon as tomorrow. I'll keep you updated on any developments regarding this issue.

Thank you!
Furby

**Reply by furby-tm** [reply]
The proverbial issue ticket can be found [here](https://github.com/orgs/wabiverse/projects/4?pane=issue&itemId=46933438) for your reference.

**Comment by SimonBenAnderson** [comment]
Thank you for the quick response and helpful feedback

**Reply by furby-tm** [reply]
@SimonBenAnderson Hey! Just a quick update, I've been refactoring some of the existing libraries a little bit and cleaning up some things, ultimately I believe it may push this upcoming release a little later into this weekend, but I wanted to be sure to let you know. Thank you!
